lib/html_to_plain_text.rb

```diff
@@ -73,19 +73,19 @@ def convert_node_to_plain_text(parent, out = '', options = {})
 
       parent.children.each do |node|
         if node.text? || node.cdata?
-          text = node.text.gsub(NON_BREAKING_SPACE, EMPTY)
+          text = node.text.gsub(NON_BREAKING_SPACE, SPACE)
```

Comment:
In this file:

We have definitions for unicode characters like `LEFT_SINGLE_QUOTE = "‘"` which we are simplifying to ASCII.

This one for `NON_BREAKING_SPACE` is the HTML escape sequence, and not the unicode character. Should we be handling the `"\u00A0"` for non-breaking spaces too?

Why are only these specific characters replaced? I assume there would be other unicode characters too, so why are these the only ones targeted?

Reply:
I am not sure why there are special rules for quotes. I assume that is part of the original library so I am not inclined to change in this PR.

I am not sure that we should be messing with any unicode characters. I think this library should simply be converting the HTML to plain text and leaving the text itself alone.